### ISSUE TITLE
Exclude go version from asset link

### ISSUE THREAD

**Issue by max-wittig** [oauth2-proxy/oauth2-proxy#634]
## Expected Behavior

Other projects just include the version of the dependency itself, so that it easily be bumped

## Current Behavior

Oauth2 proxy includes the go version, which as I user of oauth2-proxy, I don't care about.

## Possible Solution

Exclude go version from release asset

E.g. instead of 

```diff
- https://github.com/oauth2-proxy/oauth2-proxy/releases/download/v5.1.1/oauth2_proxy-v5.1.1.linux-amd64.go1.14.2.tar.gz
+ https://github.com/oauth2-proxy/oauth2-proxy/releases/download/v5.1.1/oauth2_proxy-v5.1.1.linux-amd64.tar.gz
```

## Context

E.g. when using ansible to deploy, you might have the version of oauth2-proxy in your group_vars, like this

```yaml
oauth2_proxy_version: "5.1.1"
```

which ansible uses then to complete the jinja2 template

```yaml
https://github.com/oauth2-proxy/oauth2-proxy/releases/download/v{{ oauth2_proxy_version }}/oauth2_proxy-v{{ oauth2_proxy_version }}.linux-amd64.go1.14.2.tar.gz  # the go version makes this not work
```

@dlouzan @fh1ch

**Comment by JoelSpeed** [comment]
I don't really know why we do this tbh, I know it was done before we adopted the repository from Bitly and I guess we just carried it on.

I don't really see an issue in changing it, except from it could be considered a breaking change.

@syscll @gargath I know you both have had opinions on this stuff before, any thoughts?

**Comment by max-wittig** [comment]
This should not be closed.

**Comment by syscll** [comment]
@max-wittig fixed in #733 if you want to take a look.
